notebooks/VITS_d-vector_multilingual_exemple.ipynb

```diff
@@ -0,0 +1,223 @@
+{
```

Comment:
I don't want to host notebooks anymore for model testing. They are terrible to maintain

Reply:
Deleted